messages/es-AR.json

```diff
@@ -13,6 +13,7 @@
   "editor.productSummary.displayBuyButton.option.always": "Siempre mostrar",
   "editor.productSummary.quantity-error": "No se pudo seleccionar la cantidad especificada",
   "editor.productSummary.unit": "Unidad",
-  "editor.productSummary.attachmentName": "+ {quantity}x {name}",
-  "editor.productSummary.displayBuyButton.option.hover": "Mostrar al pasar el ratón (web)"
+  "editor.productSummary.attachmentName": "{sign} {quantity}x {name}",
```

Comment:
I'm not sure the formatting should be done here. What if you want to use an svg icon for the sign? Or make the quantity bold?

Reply:
leaving this for a future PR